open-broker-common/src/main/java/de/fraunhofer/iais/eis/ids/broker/core/common/persistence/ResourceMessageHandler.java

```diff
@@ -112,6 +138,9 @@ public DefaultSuccessMAP handleValidated(ResourceMAP messageAndPayload) throws R
         try {
             //No Exception occurred. Send MessageProcessedNotificationMessage
             DefaultSuccessMAP returnValue = new DefaultSuccessMAP(infrastructureComponent.getId(), infrastructureComponent.getOutboundModelVersion(), messageAndPayload.getMessage().getId(), securityTokenProvider.getSecurityTokenAsDAT(), responseSenderAgent);
+            logger.info("Component ID :"+ infrastructureComponent.getId().toString());
```

Comment:
Same as above: Rather log the incoming message via some central message-logger function. Then, here you can print a log that the registration was successful